src/enchant.h

```diff
@@ -125,7 +125,7 @@ const char *enchant_broker_get_error (EnchantBroker * broker);
 
 /**
  * EnchantBrokerDescribeFn
- * @provider_name: The provider's identifier, such as "hunspell" or "aspell", in UTF8 encoding
+ * @provider_name: The provider's identifier, such as "nuspell" or "aspell", in UTF8 encoding
```

Comment:
Please remove this change.

Reply:
https://github.com/nuspell/enchant/commit/1739563777b88c5cb0203bb33ca77d784ca06d8b#diff-586d59cc85e33aafde5c8c1b841e7329R128